docs/tutorials/restless_measurements.rst

```diff
@@ -0,0 +1,197 @@
+    cal_drag.set_analysis_options(data_processor=standard_processor)
+
+    # enable restless mode and set override_processor_by_restless to False.
+    cal_drag.enable_restless(rep_delay=1e-6, override_processor_by_restless=False)
+
+If you run the experiment in this setting you will see that the data is often
+unusable which illustrates the importance of the data processing. As detailed
+in Ref. [2] restless measurements can be done with a wide variety
+of experiments such as fine amplitude and drag error amplifying gate sequences
+as well as randomized benchmarking.
+
+Calculating restless quantum processor speed-ups
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+We can compare the time spent by the quantum processor executing restless and
+standard jobs. This allows us to compute the effective speed-up we gain when
+performing restless experiments. Note that we do not consider any classical
+run-time contributions such as runtime-compilation or data transfer times [3].
+
+The time to run :math:`K` circuits and gather :math:`N` shots for each
+circuit is
+
+.. math::
+
+    \tau^{(x)} = NK\left(\tau^{(x)}_\text{reset}+\tau^{(x)}_\text{delay}+
+    \langle{\tau}_\text{circ}\rangle+\tau_\text{meas}\right),
+
+where :math:`\tau^{(x)}_\text{reset}` and :math:`\tau^{(x)}_\text{delay}`
+are the reset and delay times, respectively. The superscript :math:`(x)`
+indicates restless :math:`(r)` or standard :math:`(s)` measurements.
+The average duration of all :math:`K` circuits in an experiment is
+:math:`\langle{\tau}_\text{circ}\rangle=K^{-1}\sum_{k=1}^{K} \tau_{\text{circ},k}`
+where :math:`\tau_{\text{circ},k}` is the duration of only the gates in circuit
+:math:`k`. We therefore compute the quantum processor speed-up of restless
+measurements as :math:`\tau^{(\text{s})}/\tau^{(\text{r})}` which is independent
+of the number of circuits and shots.
+
+The standard reset time is on average :math:`\tau^{(s)}_\text{reset} = 4\,\mu s`
+whereas :math:`\tau^{(r)}_\text{reset} = 0\,\mu s` since we do not reset the
+qubit in a restless experiment. By default, the repetition delay is
+:math:`\tau^{(s)}_\text{delay} = 250\,\mu s`. For our restless experiments we
+set :math:`\tau^{(r)}_\text{delay} = 1\,\mu s`. These speed-ups can be evaluated
+using the code below.
+
+.. jupyter-execute::
+
+    from qiskit import schedule, transpile
+
+    dt = backend.configuration().dt
+    inst_map = backend.defaults().instruction_schedule_map
+    meas_length = inst_map.get("measure", (qubit, )).duration * dt
+
+    # Compute the average duration of all circuits
+    durations = []
+    for qc in cal_drag.circuits():
+        qc.remove_final_measurements(inplace=True)
+        qc_schedule = schedule(
+            transpile(qc, backend, initial_layout=[qubit]),
+            backend
+        )
+        durations.append(qc_schedule.duration)
+
+    tau = sum(durations) * dt / (len(durations))

```

Comment:
```suggestion
    # Remove measurement instruction
    circuits = []
    for qc in cal_drag.circuits():
        qc.remove_final_measurements(inplace=True)
        circuits.append(qc)

    # Lowering the experiment program
    executed_circs = transpile(
        cal_drag.circuits(),
        backend,
        initial_layout=[qubit],
        scheduling_method="alap",
        **cal_drag.transpile_options.__dict__,
    )
    durations = [c.duration for c in executed_circs]

    tau = sum(durations) * dt / (len(durations))
```
Use of schedule IR is kind of discouraged. Now you can use circuit scheduler pass.